client/app/telecom/pack/xdsl/modem/service/pack-xdsl-modem-service.controller.js

```diff
@@ -0,0 +1,331 @@
+angular.module('managerApp').controller('XdslModemServiceCtrl', class XdslModemServiceCtrl {
+  /* @ngInject */
+  constructor($q, $stateParams, $translate, $uibModal, OvhApiXdsl, TucPackXdslModemMediator,
+    TucToast) {
+    this.$stateParams = $stateParams;
+    this.$translate = $translate;
+    this.$q = $q;
+    this.OvhApiXdsl = OvhApiXdsl;
+    this.TucToast = TucToast;
+    this.$uibModal = $uibModal;
+    this.mediator = TucPackXdslModemMediator;
+
+    this.service = {};
+  }
+
+  getService() {
+    this.getBlocIP();
+    this.getCallWaiting();
+    this.getContentSharing();
+    this.getFTP();
+    this.getIpsecAlg();
+    this.getSipAlg();
+    this.getUPnP();
+  }
+
+  getBlocIP() {
+    return this.OvhApiXdsl.Modem().BlocIp().v6().get({
+      xdslId: this.$stateParams.serviceName,
+    }).$promise.then(({ xdslTask }) => {
+      if (xdslTask === 'enabled') {
+        this.service.blocIP = true;
```

Comment:
What's the difference between `blocIP` and `isBlocIP`  ? Shouldn't it be `ipBlock` ?

Also you can 
```suggestion
        this.service.blocIP = xdslTask === 'enabled';
```

Is there a reason why the API doesn't give a boolean ?

Reply:
The API returns two values 'enabled' or 'disabled', not boolean

Reply:
Why ? Did you ask to change it ?